metaflow/plugins/kfp/tests/sample_flows/foreach_complex_flow_3.py

```diff
@@ -0,0 +1,62 @@
+from metaflow import FlowSpec, step
+
+
+class ForeachComplexFlow3(FlowSpec):
```

Comment:
Please give the test files good names, what are they testing? complex_flow_[1-5] doesn't convey the test case it is performing.  Please update all test files accordingly.  

thanks!

Reply:
@talebzeghmi I actually put the descriptions of the tests in a Docstring style comment within the files.